code/modules/surgery/organs/tails.dm

```diff
@@ -84,6 +94,8 @@
 /obj/item/organ/tail/lizard/fake
 	name = "fabricated lizard tail"
 	desc = "A fabricated severed lizard tail. This one's made of synthflesh. Probably not usable for lizard wine."
+	tail_type = null
+	spines = null
```

Comment:
These two variables are text, and there's no instance of them being nullable before. They have the value "None", that I assume is for the same purposes you want to give them, no?

If you could turn the values into defines, that'd be a code improvement.

Reply:
This is for surgically attaching limb-grower-grown tails to people. If the type/spines are null, the attached tail will inherit the features defined in the tail receiver's DNA - otherwise, the features on the tail get applied to the receiver's features. "None" is non-null. Should I change this behavior to trigger on a value of something like "Inherit From DNA"?

Reply:
This is behavior you added, right?
Definitely turn the possible values into defines, the currently null value as well, to simulate an enum.
And document it.

Reply:
Do you want me to make every sprite accessory name a define, because that's the logical conclusion of what you seem to imply with this request.

Reply:
~~It seems that the only values `tail_type` gets are `"None"`, `"Cat"`, `"Smooth"` and `"Monkey"`. Am I missing something?~~

Alright, I see what you mean now that I look at the code a bit closer.
Just document the variables at their definition, stating that they're nullable, and how they behave when null-valued.